### DISCUSSION TITLE
RuneLite keeps loading in second monitor regardless of Windows Settings (Win11 2024)

### DISCUSSION THREAD

**Discussion by iiCelly** [runelite/runelite#17351] [Q&A] [answered]
For whatever reason, I cannot find or access the RuneLite settings to see if there is something janky, also I cannot get it to stop loading into the second monitor even though I have that monitor SET as my second monitor. It does whatever it wants... It will choose the monitor that is not my main display, I can set my other monitor to the main display and it will then go to what should be the correct monitor. Its like a + + magnetic field.

**Comment by Felanbird** [comment, marked as answer]
RuneLite settings are found in the `RuneLite` plugin in the client, playing around with the setting found in there called `Remember client position` will probably resolve your issue. (turn off/on, close/open, move client, etc)

**Reply by iiCelly** [reply]
Gonna have a look now, thanks for the heads up.

EDIT: It did work, thank you!
